src/Views/KeyboardLayoutView.vala

```diff
@@ -121,10 +153,24 @@ public class KeyboardLayoutView : AbstractInstallerView {
         }
 
         show_all ();
-    }
 
-    public void set_language (string lang) {
-        
+        Idle.add (() => {
+            string? country = Configuration.get_default ().country;
```

Comment:
Again, same thing here.

Reply:
This already is its own method, just wrapped in an `Idle.add`. It's pointless to make another method just to save a lambda.